app/models/user.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Models the logged in user
+class User
+  def initialize(remote_user:, groups: [], orcid: nil)
+    @remote_user = remote_user
+    @sunetid = remote_user.split('@').first
+    @groups = Groups.new(groups:)
+    @orcid = orcid
```

Comment:
I think this is `@login` in Hydra ETD. If we're going to be copy/pasting bunches of code over from that codebase, it might make sense to stick to its naming. On the other hand, this change makes clearer what you get (what is a login?) in which case we'll need to make targeted tweaks when we port code. NBD either way I don't think.

Reply:
I felt `login` wasn't particularly clear on what the content was and we mostly use the `sunetid` naming in other apps now. I'd lobby to go with `sunetid` so as not to maintain the legacy lack of clarity and expect devs could make the switch relatively easy.